modules/openxr/SCsub

```diff
@@ -85,3 +85,6 @@ env.modules_sources += module_obj
 
 # Needed to force rebuilding the module files when the thirdparty library is updated.
 env.Depends(module_obj, thirdparty_obj)
+
+if env["tools"]:
+    SConscript("editor/SCsub")
```

Comment:
Instead of using an `editor/SCsub` which calls `env.add_source_files(env.modules_sources, "*.cpp")`, I think it would be better to do this further up (after `env_openxr.add_source_files(module_obj, "action_map/*.cpp")`):
```
if env["tools"]:
	env_openxr.add_source_files(module_obj, "editor/*.cpp")
```

I'm not sure what the implications are exactly and your current code likely works, but I'm not sure it tracks the dependencies properly to rebuild the editor files when the thirdparty code changes, which is the purpose of the whole `module_obj` business above.

Reply:
I've moved up above the depends line, but I honestly am not sure, this part of scons I'm not very clear on, I just copy what we do in other modules